Screenbox/Pages/MainPage.xaml

```diff
@@ -333,29 +333,81 @@
+                                Stroke="#0078D7"
+                                StrokeThickness="4" />
+                        </Border>
+
+                        <Grid Grid.Column="1">
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="*" />
+                            </Grid.RowDefinitions>
+
+                            <TextBlock
+                                AutomationProperties.AccessibilityView="Content"
+                                FontWeight="SemiLight"
+                                Style="{StaticResource SubtitleTextBlockStyle}">
+                                <Run Text="For more information about this issue and possible fixes, visit" /><LineBreak />
```

Comment:
Are we not localizing this string?

Reply:
Oh absolutely. IIRC I chose not to do it because I felt it could be improved rather than making a direct copy of the BSOD.
Does the "possible fixes" part makes sense in our case? Maybe include instructions to create an issue?